libs/core/src/lib/icon-tab-bar/icon-tab-bar.component.scss

```diff
@@ -0,0 +1,154 @@
+@import '~fundamental-styles/dist/icon-tab-bar';
+@import '~fundamental-styles/dist/list.css';
+
+.cdk-drag-placeholder {
+  opacity: .3;
+  pointer-events: none;
+}
+
+// These styles should move to fundamental/styles repo.
```

Comment:
These styles are moving to sundamental/styles https://github.com/SAP/fundamental-styles/pull/2616. 
I left them here for testing during review